File: ASW-Springboot-Java/src/main/java/com/example/aswspringbootjava/domain/post/Post.java
package com.example.aswspringbootjava.domain.post;

import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@NoArgsConstructor
@Getter
@Entity
public class Post {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(length = 50, nullable = false)
    private String title;

    @Column(length = 50, nullable = false)
    private String content;

    private String author;

    @Builder
    public Post(String title, String content, String author){
        this.title = title;
        this.content = content;
        this.author = author;
    }
}

File: AWS+SpringBoot/src/main/kotlin/com/ex/aws_springboot/web/domain/posts/Post.kt
package com.ex.aws_springboot.web.domain.posts

import javax.persistence.*

@Entity
class Post(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long? = null,

    @Column(length = 500, nullable = false)
    val title: String,

    @Column(columnDefinition = "TEXT", nullable = false)
    val content: String,

    val author: String? = null
) {
}